_vcsh_bash

```diff
@@ -52,11 +52,15 @@ _vcsh_git_command () {
 }
 
 _vcsh () {
-	local cur prev words cword OPTS
+	local cur prev words OPTS
 	_init_completion -n = || return
 
-	local repos cmds
-	repos=( $(command vcsh list) )
+	local r reponames
+	local -A repos
+	mapfile -t reponames < <(command vcsh list)
+	for r in "${reponames[@]}"; do repos["$r"]="$r"; done
+	unset r reponames
```

Comment:
Just because I'm more into ZSH that Bash, any pointers on why these need explicit unsets?

Reply:
`unset` simply makes sure there is no name collision later in the function. It's a belt and suspenders thing really.

Reply:
Okay. It's moments like these that I wish shell code hygiene worked more like Rust, but there is no harm in explicitly doing your own housekeeping. Carry on.